src/control/cmd/dmg/pool.go

```diff
@@ -188,7 +189,47 @@ func (d *PoolDestroyCmd) Execute(args []string) error {
 	return err
 }
 
-// PoolQueryCmd is the struct representing the command to destroy a DAOS pool.
+// PoolReintegrateCmd is the struct representing the command to Add a DAOS target.
+type PoolReintegrateCmd struct {
+	logCmd
+	connectedCmd
+	UUID    string `long:"pool" required:"1" description:"UUID of the DAOS pool to start reintegration in"`
+	Rank    uint32 `long:"rank" required:"1" description:"Rank of the targets to be reintegrated"`
+	Targets string `long:"idx" required:"1" description:"list of idx(s) to be reintegrated into the rank"`
```

Comment:
My reasoning behind this is that when rebuild is running and completes it tells you which targets were excluded by their rank and idx.

so rebuild will generate output logs such as:
`daos_io_server:0 Target (rank 5 idx 0) is down.`
and 
`daos_io_server:0 Target (rank 5 idx 0) is excluded.`

Doing it this way also simplifies things internally because the  functions to set the target status to up expects a component address list which are rank and idx pairs, as opposed to target ID numbers.

I agree that naming the field "Target" is confusing and I should probably rename it idxs

Reply:
Maybe TargetIdx would make it clearer what an idx is?